pkgs/by-name/ma/mako/package.nix

```
@@ -14,6 +27,7 @@ stdenv.mkDerivation (finalAttrs: {
     sha256 = "sha256-QtYtondP7E5QXLRnmcaOQlAm9fKXctfjxeUFqK6FnnE=";
   };
 
```

```suggestion

  strictDeps = true;
```

which is already implicitly set for cross-comp, but this will help enforce correct dependency specification when build == host platform